lib/user-interface/app/src/pages/profile/InviteQrOverlay.css

```
@@ -0,0 +1,104 @@
+  width: min(90vw, 62vh);
+  height: auto;
+  max-width: 100%;
+  border-radius: 8px;
+  /* qrcode.react paints its own white background path across the quiet zone,
+     so the card needs no extra padding to preserve it. */
+  background-color: #FFFFFF;
+}
+
+.invite-qr-hint {
+  margin: 0;
+  font-family: 'akzidenz-grotesk-next-pro', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "akzidenz-grotesk-next-pro" (font-family-name-quotes)

(font-family-name-quotes)